src/referencePackageSourceGenerator/GenerateSource/targetPackage.csproj.template

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>##Tfm##</TargetFramework>
+    <##TfmAttr##>##Tfm##</##TfmAttr##>
```

Comment:
I find it a bit odd to refer to this as an attribute in the scope of an msbuild project defined in xml.  Consider naming it `TfmProperty` or `TfmPropertyName`

Reply:
Sure thing, renamed to `TfmPropertyName`